src/main/resources/application.properties

```diff
@@ -1 +1,7 @@
 spring.application.name=LoanManagerAPI
+spring.datasource.url=jdbc:postgresql://localhost:5432/loan_db
```

Comment:
I think we will use mySQL instead, since our data is structured

spring.datasource.driver-class-name=com.mysql.cj.jdbc.Driver
spring.datasource.url=jdbc:mysql://localhost:3306/user
spring.datasource.username=root
spring.datasource.password=Password
spring.jpa.properties.hibernate.dialect=org.hibernate.dialect.MySQL8Dialect
spring.jpa.show-sql=true